floss/identify.py

```diff
@@ -173,11 +174,14 @@ def find_decoding_function_features(vw, functions, disable_progress=False) -> Tu
                 function_data["features"].append(feature)
 
             function_data["score"] = get_function_score_weighted(function_data["features"])
-
-            logger.debug("analyzed function 0x%x - total score: %.3f", function_address, function_data["score"])
-            for feat in function_data["features"]:
-                logger.trace("  %s", feat)
```

Comment:
why remove these messages?

Reply:
We remove this because it primarily focuses on logging detailed information about analyzed functions, including their scores and the features identified within them. 

- But by creating a summary list that includes both the scores and call counts, the code extends its functionality to 
   provide a concise overview of each decoding candidate function.
- Also the new list can be returned from a function, stored, passed to other parts of the code, or even exported to a file
- It makes the output more accessible and the code clearer and more maintainable.

@williballenthin KIndly refer to the new changes made in the pr and also suggest me if any changes are required .

Reply:
let's keep them